lib/bolt/project_manager.rb

```diff
@@ -119,7 +119,7 @@ def create(path, name, modules)
         @outputter.stop_spin
       end
 
-      data = { 'name' => project_name }
+      data = { 'name' => project_name, 'profiles' => [] }
```

Comment:
Should be `policies`

Reply:
Removed this altogether. Per a discussion in the design doc, we felt that while this feature is experimental we should not add it to projects by default.

Reply:
Oh yeah that makes way more sense 👍